pkg/kubeapiserver/admission/initializer.go

```diff
@@ -35,6 +36,7 @@ type PluginInitializer struct {
 	cloudConfig        []byte
 	restMapper         meta.RESTMapper
 	quotaConfiguration quota.Configuration
+	resourceFilter     resourcefilter.Interface
```

Comment:
Maybe we should either go with resourceFilter or handling within particular plugins like VAP as what we did here: https://github.com/kubernetes/kubernetes/blob/8c80c07e85833c5b426fdd774d2729733d14adb6/staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/generic/plugin.go#L189
Maintaining two may cause confusion.